apphelpers/db/peewee.py

```diff
@@ -72,14 +84,8 @@ def dbtransaction(db):
     def wrapper(f):
         @wraps(f)
         def f_wrapped(*args, **kw):
-            if not db.in_transaction():
-                with db.connection_context():
-                    with db.atomic():
-                        result = f(*args, **kw)
-            else:
-                with db.atomic():
-                    result = f(*args, **kw)
-            return result
+            with dbtransaction_ctx(db):
+                return f(*args, **kw)
```

Comment:
We won't be using it in FastAPI. This is only for hug compatibility (and apphelpers tests).